docs/reference/api/generated-api-html/index.html

```diff
@@ -8285,13 +8285,15 @@ <h3 class="field-label">Example data</h3>
         "fieldName" : [ "fieldName", "fieldName" ],
         "addField" : { },
         "transformType" : "add_field",
-        "removeField" : { }
+        "removeField" : { },
+        "isBreaking" : true
```

Comment:
this was automatically generated and I'm not sure what it is or whether I should change it